t/from-hash.t

```diff
@@ -43,7 +43,7 @@ for my $string (10, '>= 2, <= 9, != 7') {
 
 SKIP: {
   skip "Can't tell v-strings from strings until 5.8.1", 1
-    unless $] gt '5.008';
+    unless $] > 5.008;
```

Comment:
Or we get rid of the version comparison entirely, this dist isn't working on 5.6 anymore since 2.141 (April 2023)

Reply:
Shall we bump it to 5.8.1 ?